assets/stylesheets/ansi-colors.scss

```diff
@@ -16,3 +16,11 @@
 .ansi-blue-fg    { color: $blue; }
 .ansi-magenta-fg { color: $pink; }
 .ansi-cyan-fg    { color: $cyan; }
+
+@media (prefers-color-scheme: dark) {
+    .ansi-white-fg   { color: $default-font-color-darktheme; }
+    .ansi-black-fg   { color: $default-font-color-darktheme; }
+    .ansi-red-fg     { color: $red; }
+    .ansi-yellow-fg  { color: $yellow; }
+    .ansi-green-fg   { color: $green; }
+}
```

Comment:
setup the editor to not produce this "glitches" in the files please

Reply:
What do you mean by 'glitches'?

Reply:
I added newlines at the end of every file if that helps.